contracts/patterns/Payable.sol

```diff
@@ -1,8 +1,7 @@
 // SPDX-License-Identifier: MIT
-
 pragma solidity >=0.6.0 <0.9.0;
 
-import "../interfaces/IERC20.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
Could we also remove the extra file for the rest of openzeppelin contracts?
- ERC165
- Initializable
- Ownable
- ReentrancyGuard

Reply:
Or if we want to keep a file for each of the contracts from openzeppelin, do the same for IERC20.sol